package.json

```diff
@@ -0,0 +1,65 @@
+{
+  "name": "foreman_leapp",
+  "version": "0.0.1",
+  "description": "Plugin for RHEL upgrades using Foreman.",
+  "main": "index.js",
+  "directories": {
+    "test": "test"
+  },
+  "scripts": {
+    "test": "node node_modules/.bin/jest webpack",
+    "lint": "./node_modules/.bin/eslint -c .eslintrc webpack/"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/oamg/foreman_leapp.git"
+  },
+  "author": "LEAPP team + The Foreman team",
+  "license": "GPL-3.0",
+  "bugs": {
+    "url": "https://projects.theforeman.org"
+  },
+  "homepage": "https://github.com/oamg/foreman_leapp",
+  "peerDependencies": {
+    "@theforeman/vendor": ">= 3.3.2"
+  },
+  "devDependencies": {
+    "@babel/core": "^7.7.0",
+    "@theforeman/env": "^3.3.2",
+    "@theforeman/builder": "^4.0.2",
+    "@theforeman/vendor-dev": "^3.3.2",
```

Comment:
can we use newer versions of @theforeman packages, while all of them point to the same version?